src/sbt-test/docker/file-permission/build.sbt

```diff
@@ -79,9 +81,10 @@ lazy val root = (project in file("."))
           |RUN ["chmod", "-R", "u=rwX,g=rwX", "/opt/docker"]
           |
           |FROM fabric8/java-centos-openjdk8-jdk
-          |RUN id -u daemon || useradd --system --create-home --uid 1001 --gid 0 daemon
+          |USER root
+          |RUN id -u demiourgos728 2> /dev/null || useradd --system --create-home --uid 1001 --gid 0 demiourgos728
           |WORKDIR /opt/docker
-          |COPY --from=stage0 --chown=daemon:root /opt/docker /opt/docker
+          |COPY --from=stage0 --chown=demiourgos728:root /opt/docker /opt/docker
```

Comment:
Any particular reason for this name :laughing:

Reply:
In Platonism, dēmiourgos (δημιουργός) meaning craftsman, was an artisan-like figure responsible for creating / maintaining the physical universe. At some point this word got misread by someone around 350AD, got mixed up with 'daimon', and out of thin air became Demogorgon. https://twitter.com/eed3si9n/status/1094012513627394049
728 is 'sbt' spelled using telephone keypad. Hopefully this is unique enough name.